Unix/tests/wsman/test_NumberProvider.cpp

```diff
@@ -1360,8 +1360,8 @@ NitsEndTest
 
 NitsTestWithSetup(TestInvokeSmallNumber_InvalidFn, NumberProvTestSetup)
 {
-    // trying to non-exisiting function
-    //omicli iv test/cpp X_SmallNumber FunciotnThatDoesnotExist
+    // trying to refer to nonexistent function
```

Comment:
this is the closest to a grammar change i made, the line didn't make sense and this makes it closer to other lines in the file